frontend/packages/policy-editor/src/types/index.ts

```diff
@@ -18,6 +19,45 @@ export interface PolicySubject {
   subjectDescription: string;
 }
 
+export interface PolicyAccessPackage {
```

Comment:
Can we use `type` instead of `interface`? 😄 Goes for entire file

```suggestion
export type PolicyAccessPackage = {
```